crytic_compile/platform/solc.py

```diff
@@ -60,7 +60,13 @@ def export_to_solc(crytic_compile: "CryticCompile", **kwargs: str) -> Union[str,
     # Create additional informational objects.
     sources = {filename: {"AST": ast} for (filename, ast) in crytic_compile.asts.items()}
     source_list = [x.absolute for x in crytic_compile.filenames]
-    source_list.sort()  # needed for Echidna, see https://github.com/crytic/crytic-compile/issues/112
+
+    # needed for Echidna, see https://github.com/crytic/crytic-compile/issues/112
+    first_source_list = list(filter(lambda f: "@" in f, source_list))
```

Comment:
Can you explain what is happening here?
Is it to order first the dependencies, and then the other files?